### ISSUE TITLE
Include tailwind support

### ISSUE THREAD

**Issue by knd775** [dprint/dprint-plugin-prettier#43]
Can you please add `prettier-plugin-tailwindcss` to this package? Thanks!

**Comment by yashpandit** [comment]
> Can you please add `prettier-plugin-tailwindcss` to this package? Thanks!

+1. would love to have this plugin 🙏🏽

**Comment by sgnilreutr** [comment]
Yes, I would love this one as well!

+1

**Comment by tomdavidson** [comment]
Im using tailwind too. Does the prettier-plugin search the prettier plugins directory?

**Comment by knd775** [comment]
> Im using tailwind too. Does the prettier-plugin search the prettier plugins directory?

No, it does not. Prettier plugins need to be bundled with this plugin in order to work.

**Comment by tomdavidson** [comment]
Thanks knd775 .

Based on the fairly recent jsdoc change #27, it doesnt look too hard but I dont quit understand some parts. Know of a doc or blog post breaking it down - contrib to dprint-plugin-prettier for dummies?

**Comment by arielvieira** [comment]
+1 for this

**Comment by vieko** [comment]
+1 for this, please!

**Comment by J3m5** [comment]
This is the last thing preventing me from adopting dprint. 
Is there any advance on this plugin?
Thank you!

**Comment by karyanayandi** [comment]
+1

**Comment by lucasdu4rte** [comment]
+1 pleasee!!

**Comment by liuhq** [comment]
I build a plugin for tailwindcss (however, more testing is needed)

[liuhq/dprint-plugin-tailwindcss](https://github.com/liuhq/dprint-plugin-tailwindcss)
